documentation/high-avail/ha-api-hld.md

```diff
@@ -283,6 +316,9 @@ typedef struct _sai_ha_scope_event_data_t
     /** Flow version */
     sai_uint32_t flow_version;
 
+    /** HA role */
+    sai_dash_ha_state_t ha_state;
+
```

Comment:
I think we are missing the a few event notifications - such as "brain split detected" and etc.

Reply:
added split_brain_detected event